app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    caching_result = {}
+
+    def wrapper(*args) -> Callable:
```

Comment:
```suggestion
    def wrapper(*args, **kwargs) -> Callable:
```

Reply:
Sorry, but https://github.com/mate-academy/py-cache-decorator/pull/947#discussion_r1468450859

Reply:
Sorry, Decorator cache creating for decorating only functions that take immutable arguments.